front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -83,7 +83,7 @@ export default function AssistantNew({
   const [planLimitReached, setPlanLimitReached] = useState<boolean>(false);
   const sendNotification = useContext(SendNotificationsContext);
 
-  const [helpDrawer, setHelpDrawer] = useState<boolean>(false);
+  const [showHelpDrawer, setShowHelpDrawer] = useState<boolean>(false);
```

Comment:
We tend to use `isXXX` to reflect boolean states, don't we 😛

Reply:
True. Will update